scripts/asm-installer/release_asm_installer

```diff
@@ -113,6 +112,12 @@ check_tags() {
   fi
 }
 
+get_stable_version() {
+  local VER; VER="$(./"${SCRIPT_NAME}" --version || true)"
+
+  echo ${VER//+/-}
```

Comment:
Quote the ${}: https://github.com/koalaman/shellcheck/wiki/SC2046

Reply:
Done. Will review the shellcheck rules comprehensively.